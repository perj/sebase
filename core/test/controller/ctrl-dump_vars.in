dump_vars?foo=bar&bar=hej