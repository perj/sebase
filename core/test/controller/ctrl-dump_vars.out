Vars:
bar -> hej
foo -> bar
